site/en/guide/function.ipynb

```diff
@@ -14,7 +14,7 @@
       "cell_type": "code",
       "execution_count": 0,
       "metadata": {
-        "cellView": "both",
+        "cellView": "form",
```

Comment:
Why is the cellview changing here?

Reply:
This is what hides the license text and someone must have left it open. Another victory for nbfmt!